src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -185,8 +185,24 @@
     <PlatformManifestFileEntry Include="icudt_no_CJK.dat" IsNative="true" />
     <PlatformManifestFileEntry Include="icudt_CJK.dat" IsNative="true" />
     <PlatformManifestFileEntry Include="icudt_EFIGS.dat" IsNative="true" />
+    <PlatformManifestFileEntry Include="icudt_efigs_locales.dat" IsNative="true" />
```

Comment:
Why is the above file named:

`icudt_EFIGS.dat`  (note the casing)

and the new file(s) named:

`icudt_efigs_locales.dat` ?

Can we be consistent in the casing?

Reply:
So currently using the different casing to differentiate between the original shard `icudt_EFIGS.dat` which includes all features for the EFIGS locales versus newer divisions by locale and culture. It comes down to how the icu_dictionary.json is [generated](https://github.com/dotnet/icu/blob/f29ba418de0d458b1d6e95caa74c056ab0356c09/icu/icu4c/source/python/icutools/databuilder/__main__.py#L272). There could definitely be a better way of distinguishing between the two but I was hesitant to change the existing names.